### DISCUSSION TITLE
MPC Alpha - What exactly is meant by the "Alpha"?

### DISCUSSION THREAD

**Discussion by RomansWorks** [Web3Auth/Web3Auth#1282] [Help]
Hi, 

Just to understand - the MPC SDK mentions Alpha and an alpha letter next to it. Is that a maturity specifier or the name of the SDK? 

If it's a maturity specifier, then what exactly is mean by the SDK being in Alpha:
1. Is it indicating that only the SDK is in an alpha stage or that the whole Web3Auth MPC solution (protocol, backends, guides, etc.) is in alpha stage?
2. What is still missing, and is there a roadmap for a stable release that we can see?
3. Is the Web3Auth MPC solution stable enough for production apps? 

Thanks

**Comment by YZhenY** [comment]
Hey roman, 

Thanks for the interest, thats the maturity specifier of the SDK.


1 & 2
Web3Auth's MPC self serve SDK solution is in Alpha at the moment. As for custom solutions the we have several enterprises using it in production and more building on the closed beta. We'll be unveiling our self-serve into beta or perhaps straight to prod in a couple of weeks.

3
If you're only planning prod in 5 weeks or longer feel free to play around with the Alpha (though it is a bit rough at the moment)

Alternatively, I would recommend starting off with Web3Auth self-host/core kit for both a shorter timeline and the ability to then migrate over quite easily. The UI/UX around it and the API that will be coming out is almost exactly the same.

**Reply by RomansWorks** [reply]
@YZhenY thanks for that, I would like to understand the options you mentioned better then:

1. How could we set up a discussion regarding the enterprise solution or a custom solution? We do have some unique requirements we're building around.

2. Self-Host core, as far as I can tell, is not MPC, is that correct? Also, just to verify my understanding - in the MPC solution a single key that can sign a transaction on its own is never present on any single device at any time; while with the self-host and the legacy solution there's a point in time where the private key is assembled at the client side and hence may be compromised on the client side?
